packages/calcite-components/src/components/dialog/readme.md

```diff
@@ -0,0 +1,156 @@
+# calcite-dialog
```

Comment:
Would this readme need to be refactored for `dialog`?

Reply:
It should be auto generated at some point.